nightingale_utils/src/utils.py

```diff
@@ -69,25 +69,76 @@ def __init__(self):
 
     def robot_configuration_lookup(self, req):
         ret = RobotConfigurationLookupResponse()
-        ret.jnt_state.header.stamp = rospy.Time.now()
-        body_part_string = {
-            req.LEFT_ARM: "left_arm",
-            req.RIGHT_ARM: "right_arm",
-            req.TORSO: "torso",
-            req.HEAD: "head",
-        }[req.body_part]
-
-        if req.configuration not in self.configurations.keys():
-            return None
-        configuration = self.configurations[req.configuration]
-
-        if body_part_string not in configuration.keys():
-            return None
-
-        ret.jnt_state.name = configuration[body_part_string]["names"]
-        ret.jnt_state.position = configuration[body_part_string]["joints"]
-        ret.jnt_state.velocity = [0.0 for jnt in ret.jnt_state.name]
-        return ret
+        if req.lookup_cart == True:
```

Comment:
I would suggest removing req.lookup_cart. its a lot easier to just fill the jnt_state and cart_state response values and let the user decide what to do with the response rather than requiring them to declare in the request which values they need